<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>numeroParImpar</class>
 <widget class="QDialog" name="numeroParImpar">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>240</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="resultParImpar">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>170</y>
     <width>249</width>
     <height>34</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>60</y>
     <width>251</width>
     <height>91</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="2" column="0">
     <widget class="QLineEdit" name="txtNum"/>
    </item>
    <item row="3" column="0">
     <widget class="QPushButton" name="pbAceptarNumParImpar">
      <property name="text">
       <string>Aceptar</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Ingrese un numero para saber si es PAR o IMPAR:</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>10</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>NUMERO PAR O IMPAR</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
